src/styles/blocks/header.scss

```diff
@@ -0,0 +1,78 @@
+.header {
+    display: inline-flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    height: $header-height;
```

Comment:
The SCSS variable `$header-height` is used here but not defined. Ensure that it is defined to avoid compilation errors.